pkg/parser/uri_test.go

```diff
@@ -0,0 +1,68 @@
+package parser
+
+import (
+	"testing"
+)
+
+type tests struct {
+	dsn    string
+	scheme string
+	path   string
+	query  string
+}
+
+var fTests = []tests{
+	{"", "", "", ""},
+	{"file:///some/path/to/file.csv?format=csv", "file", "/some/path/to/file.csv", "format=csv"},
+	{"file:///some/path/to/file.csv", "file", "/some/path/to/file.csv", ""},
+	{"os:stdout?format=csv", "os", "", "format=csv"},
```

Comment:
Should the path not be `stdout` in this case?

Reply:
Fixed.